pkg/api/v1beta3/dynakube/extensions.go

```diff
@@ -102,3 +104,45 @@ type ImageRefSpec struct {
 	// Indicates a tag of the image to use
 	Tag string `json:"tag,omitempty"`
 }
+
+type LogAgentSpec struct {
```

Comment:
`Templates` section is a container for different features now.

I think `LogAgentSpec` struct should be part of log_monitoring_types.go . 
`TemplatesSpec` struct could be moved to dynakube_types.go .

Reply:
done in https://github.com/Dynatrace/dynatrace-operator/pull/3595/commits/3072f91d4046c25f070d99d2d5415ec4b331a85b